deployments/charts/kuma/templates/cp-additional-k8s-secrets.yaml

```diff
@@ -0,0 +1,16 @@
+{{ range $secret := .Values.controlPlane.additionalK8sSecrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ tpl $secret.name $ | quote }}
+  labels:
+    {{- include "kuma.labels" $ | nindent 4 }}
+    app: {{ include "kuma.name" $ }}-control-plane
+    {{ range $key, $value := $.Values.controlPlane.extraLabels }}
+    {{ $key | quote }}: {{ $value | quote }}
+    {{ end }}
+stringData:
```

Comment:
I think for now `stringData` will be sufficient for most people. Basically it just makes Kubernetes automatically base64 encode the data. I think it's easier to use for most people because they won't have to base64 encode data on their own.